web/src/beta/features/Editor/tabs/map/RightPanel/LayerInspector/FeatureData.tsx

```diff
@@ -1,23 +1,86 @@
-import JsonView from "react18-json-view";
+import { useCallback, useEffect, useState } from "react";
 import "react18-json-view/src/style.css";
 import "react18-json-view/src/dark.css";
+import JsonView from "react18-json-view";
+import { v4 as uuidv4 } from "uuid";
 
+import SidePanelSectionField from "@reearth/beta/components/SidePanelSectionField";
 import Text from "@reearth/beta/components/Text";
+import { GeoJsonFeatureUpdateProps } from "@reearth/beta/features/Editor/useSketch";
 import { Geometry } from "@reearth/beta/lib/core/engines";
+import { Feature } from "@reearth/beta/lib/core/mantle";
 import { useT } from "@reearth/services/i18n";
 import { styled } from "@reearth/services/theme";
 
+import { FieldComponent } from "./CustomPropertField";
+
 type Props = {
   selectedFeature?: {
     id: string;
     geometry: Geometry | undefined;
     properties: any;
   };
+  isSketchLayer?: boolean;
+  customProperties?: any;
+  layerId?: string;
+  sketchLayerFeature?: Feature;
+  onGeoJsonFeatureUpdate?: (inp: GeoJsonFeatureUpdateProps) => void;
+};
+
+export type ValueProp = string | number | boolean | undefined;
+export type FieldProp = {
+  id: string;
+  type: string;
+  title: string;
+  value?: ValueProp;
 };
 
-const FeatureData: React.FC<Props> = ({ selectedFeature }) => {
+const FeatureData: React.FC<Props> = ({
+  selectedFeature,
+  isSketchLayer,
+  customProperties,
+  layerId,
+  sketchLayerFeature,
+  onGeoJsonFeatureUpdate,
+}) => {
   const t = useT();
 
+  const [field, setField] = useState<FieldProp[]>([]);
+
+  useEffect(() => {
+    if (!customProperties) return;
+    const entries = Object.entries<string>(customProperties);
+    const sortedValues = entries
+      .map(([key, value]) => ({ key, value }))
+      .sort((a, b) => {
+        const aIndex = parseInt(a.value.split("_")[1]);
+        const bIndex = parseInt(b.value.split("_")[1]);
+        return aIndex - bIndex;
+      });
+
+    const fieldArray = sortedValues.map(({ key, value }) => ({
+      id: uuidv4(),
+      type: value.replace(/_\d+$/, ""),
+      title: key.toLowerCase(),
```

Comment:
Why do we need to use lower case? I think the title should present what we have in data.